mobile-first.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bezalell Portfolio</title>
+  </head>
+  <body>
+    <!-- navbar -->
+    <header>
+      <nav class="navbar">
+        <div class="name-container">
+          <a href="index.html">
+            <h2>Bezalell</h2>
+          </a>
+        </div>
+        <div>
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
+      </div>
+      <div class="links-container">
+        <ul>
+          <li>
+            <a>Portfolio</a>
+          </li>
+          <li>
+            <a>About</a>
+          </li>
+          <li>
+            <a>Contact</a>
+          </li>
+          <li>
+            <a
+            ><img
+            src="images/Normal Button/Tertiary/Disabled.png"
+            alt="Message"
+            /></a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <!-- header -->
+    <section class="header">
+      <div class="welcome">
+        <h1>
+          Hey There. <span>&#39;</span>I' am Bezalell
+        </h1>
```

Comment:
1. The heading has a left and right `margin` as `24px`. h1 text seems to be on the left. Also `I` of `I' am Bezalell` is on the first line.
2. `<span>&#39;</span>` could only be `&#39;`

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/220967377-32cc75ce-b572-4246-9ce4-c480142fd590.png)| ![image](https://user-images.githubusercontent.com/61361037/220967254-cdc1950d-0d2b-4cb1-940a-d72535689ed3.png)|